spec/features/venues/create_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Venues', type: :feature do
```

Comment:
I think the 'Venues' feature string should be rewritten to specify "create a Venue", delete a venue, update a venue, etc if you're using separate files to test the separate CRUD (Create, Read, Update, Destroy) actions. This is relevant to the other 2 venues feature spec files, too.